engine-src/Game/LambdaHack/Client/UI/HandleHelperM.hs

```diff
@@ -196,7 +202,7 @@ pickLeaderWithPointer = do
   K.PointUI x y <- getsSession spointer
   let (px, py) = (x `div` 2, y - K.mapStartY)
   -- Pick even if no space in status line for the actor's symbol.
-  if | py == rheight - 2 && px == 0 -> memberBack True
+  if | py == rheight - 2 && px == 0 -> memberCycleLevel True Forward
```

Comment:
I missed this one: I think the `memberCycle` command could be more useful to mouse users here (this is what happens when you click on the star in the bottom left corner).